src/ORM/Behavior/TranslateBehavior.php

```diff
@@ -110,16 +113,17 @@ public function initialize(array $config)
      * @param string $model the model field value
      * @return void
      */
-    public function setupFieldAssociations($fields, $table, $model)
+    public function setupFieldAssociations($fields, $table, $model, $strategy)
     {
+        $targetAlias = Inflector::slug($table);
+        $alias = $this->_table->alias();
```

Comment:
Yeah it should, that `$alias` is just the alias of the model who has the behavior attached, not the one doing the translating.